pyccel/stdlib/external/dfftpack.py

```diff
@@ -1,3 +1,12 @@
+#------------------------------------------------------------------------------------------#
+# This file is part of Pyccel which is released under MIT License. See the LICENSE file or #
+# go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details.     #
+#------------------------------------------------------------------------------------------#
+
+"""
+Module exposing the dfftpack library functions to pyccel
```

Comment:
It would be good to add a link. @saidctb is this the correct link: https://www.netlib.org/fftpack/ ? I can't find `dfftpack`, only `fftpack`

Reply:
yes